integrations/shopify/src/actions/get-customerOrders.ts

```diff
@@ -0,0 +1,33 @@
+import { SHOPIFY_API_VERSION } from '../const'
+import { prepareParams } from '../utils'
+import axios from 'axios'
+import * as botpress from '.botpress'
+
+type GetCustomerOrders = botpress.IntegrationProps['actions']['getCustomerOrders']
+
+export const getCustomerOrders: GetCustomerOrders = async ({ ctx, input, logger }) => {
+  const { customer_id, status } = input
+  const filters = await prepareParams({ customer_id, status })
```

Comment:
I was using something else; then Fleur was helping me with something else and suggested this way. I was looking for a way to take all the parameters and return them as query strings to be attached to the URL directly.

Reply:
I think simply this would work
 ```ts
 const response = await axios.get(
      `/admin/api/${SHOPIFY_API_VERSION}/orders.json${qs.stringify(input)}`
    )
```